kubernetes/plugin.go

```diff
@@ -121,6 +122,13 @@ func pluginTableDefinitions(ctx context.Context, d *plugin.TableMapData) (map[st
 }
 
 func listK8sDynamicCRDs(ctx context.Context, cn *connection.ConnectionCache, c *plugin.Connection) ([]v1.CustomResourceDefinition, error) {
+	// get the crds from config if any
+	kubernetesConfig := GetConfig(c)
+	filterCrds := kubernetesConfig.CustomResourceTables
+	if len(filterCrds) == 0 {
+		return nil, nil
+	}
+
```

Comment:
Do we need this change?

Reply:
Yes, @misraved, we need this code block to validate if a list of custom resource tables is set in the config before creating the client.